src/test/java/com/puppycrawl/tools/checkstyle/api/AbstractCheckTest.java

```diff
@@ -73,4 +78,65 @@ public void testVisitToken() {
         // Eventually it will become clear abstract method
         check.visitToken(null);
     }
+
+    @Test
+    public void testGetLine() throws Exception {
+        final AbstractCheck check = new AbstractCheck() {
+            @Override
+            public int[] getDefaultTokens() {
+                return CommonUtils.EMPTY_INT_ARRAY;
+            }
+
```

Comment:
Please remove these empty lines before close `}`. There are a few in tests you have made.